migrations/232_enum_pogil_role__create.sql

```diff
@@ -0,0 +1,7 @@
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT * FROM pg_type WHERE typname = 'enum_pogile_role') THEN
```

Comment:
```suggestion
    IF NOT EXISTS (SELECT * FROM pg_type WHERE typename = 'enum_pogil_role') THEN
```

Is this confirmed to work locally? If you were to run this locally without the migration (by say, starting the docker container and opening postgres with `psql`), and then apply the migration, does it work okay?

Reply:
For this migration, I had copied from another migration [008_enum_instance_question_status__create.sql](https://github.com/PrairieLearn/PrairieLearn/blob/master/migrations/008_enum_instance_question_status__create.sql) that also added a new enum type. Just to check that it worked, I ran the docker container without migrations and using `psql`, created the enum (and ran again just to make sure it exists):

```
postgres=# CREATE TYPE enum_pogil_role AS ENUM ('None', 'Manager', 'Recorder', 'Reflector', 'Contributor');
CREATE TYPE

postgres=# CREATE TYPE enum_pogil_role AS ENUM ('None', 'Manager', 'Recorder', 'Reflector', 'Contributor');
ERROR:  type "enum_pogil_role" already exists
```

We also ran the `233` migration to manually add the enum as a column to the `group_users` table:
```
postgres=# ALTER TABLE group_users ADD COLUMN pogil_role enum_pogil_role DEFAULT 'None'::enum_pogil_role;
ALTER TABLE
postgres=# \d group_users
                          Table "public.group_users"
   Column   |      Type       | Collation | Nullable |         Default
------------+-----------------+-----------+----------+-------------------------
 group_id   | bigint          |           | not null |
 user_id    | bigint          |           | not null |
 pogil_role | enum_pogil_role |           |          | 'None'::enum_pogil_role
Indexes:
    "group_users_pkey" PRIMARY KEY, btree (group_id, user_id)
    "group_users_group_id_key" btree (group_id)
    "group_users_user_id_key" btree (user_id)
Foreign-key constraints:
    "group_users_group_id_fkey" FOREIGN KEY (group_id) REFERENCES groups(id) ON UPDATE CASCADE ON DELETE CASCADE
    "group_users_user_id_fkey" FOREIGN KEY (user_id) REFERENCES users(user_id) ON UPDATE CASCADE ON DELETE CASCADE
```